packages/api-server/src/db/constant.ts

```diff
@@ -1,4 +1,8 @@
-export const MAX_QUERY_NUMBER = 10000;
-export const MAX_QUERY_TIME_MILSECS = 1000 * 10; // 10 seconds
+import { envConfig } from "../base/env-config";
```

Comment:
pls remove this line. you can keep `DEFAULT_xxx` value here, and use the (envConfig value/ default value) in the business logical code.